test/test_autograd.py

```diff
@@ -4728,6 +4728,33 @@ def test_power_function(self):
         c.backward()
         self.assertEqual(b.grad, torch.tensor([-inf, 0., 0.]))
 
+    def test_float_power_function(self):
```

Comment:
These tests are cool but I don't think they're necessary, and they can be removed from this PR.

Although this is confusing (and we're working on changing this), the entries this PR adds to common_methods_invocations.py are a sufficient autograd test, I think.

Reply:
Gotcha! The redundant test case has been removed!